### PR TITLE
Add player2

### PR DESCRIPTION
**Pull request by lrharris215**
The game now takes in a Players array. The first player can make 1 mark on the board, and then the second player can make 1 mark on the board. The game switches the active player after every turn.

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -22,9 +23,21 @@ def start
...
+      print_board
+      @board.place_mark(@active_player.mark, @active_player.get_input(@display))
+      print_board
+      switch_player
+      @board.place_mark(@active_player.mark, @active_player.get_input(@display))
+      print_board
+      @game_over = true
+    end
+  end
+
```
Comment (kudukacar) [inline lib/game.rb:36]: I don't think you need a while loop now or the game_over flag.  All you need at this stage is print_board, place_mark, print_board, place_mark, print_board

File: lib/game.rb [hunk 2]
```diff
@@ -22,9 +23,21 @@ def start
...
+  private
+
+  def switch_player
+    @active_player = @active_player === @player_1 ? @player_2 : @player_1
   end
```
Comment (kudukacar) [inline lib/game.rb:42]: Do you need this now?  Each player is taking only one turn.  The first place_mark gets the first_player as an argument and the second place_mark get the second_player as an argument.  Remember the bowling game kata - don't get ahead of yourself in terms of design.

Comment (lrharris215) [reply lib/game.rb:42]: Yeah I wasn't sure yesterday how many turns I wanted it to run for. I commented everything out.

Comment (kudukacar) [reply lib/game.rb:42]: Also be careful about commenting out code.  Commented out code quickly becomes stale.  And, you don't want to lock yourself to a particular design.

Comment (lrharris215) [reply lib/game.rb:42]: That's true. But I am reasonably sure I will at least need the switch_players function very soon.

**Pull request merged**